core/domain/user_domain_test.py

```diff
@@ -0,0 +1,364 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for user domain objects."""
+
+from core.domain import user_domain
+from core.tests import test_utils
+import feconf
+
+
+class UserGlobalPrefsTests(test_utils.GenericTestBase):
+    """Test domain object for user global email preferences."""
+
+    def setUp(self):
+        super(UserGlobalPrefsTests, self).setUp()
+        self.test_prefs_email = (user_domain.UserGlobalPrefs(
+            True, False, True, False))
+        self.default_prefs_email = (
+            user_domain.UserGlobalPrefs.create_default_prefs())
+
+    def test_initialization(self):
+        """Testing init method."""
+        receive_email_updates = (
+            self.test_prefs_email.can_receive_email_updates)
+        receive_editor_role_email = (
+            self.test_prefs_email.can_receive_editor_role_email)
+        receive_feedback_message_email = (
+            self.test_prefs_email.can_receive_feedback_message_email)
+        receive_subscription_email = (
+            self.test_prefs_email.can_receive_subscription_email)
+
+        self.assertEqual(True, receive_email_updates)
```

Comment:
For checking True or False, please use `self.assertTrue(..)` and `self.assertFalse(..)`. For reference, please these functions in the unittest module's documentation [here](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertTrue).